src/app/page.tsx

```diff
@@ -212,14 +214,34 @@ export default function Home() {
             onChange={(e) => setMessage(e.target.value)}
             onKeyDown={handleKeyPress}
             disabled={isLoading}
+            InputProps={{
+              sx: {
+                bgcolor: "black",
+                color: "white",
+                borderRadius: "5px",
+              },
+            }}
+            InputLabelProps={{
+              sx: {
+                color: "white",
+              },
+            }}
           />
-          <Button
-            variant="contained"
+          <IconButton
+            color="primary"
             onClick={() => sendMessage(message)}
             disabled={isLoading}
+            sx={{ 
+              bgcolor: "#B8860B", 
+              color: "black", 
+              transition: "transform 0.2s",
+              "&:hover": {
+                transform: "scale(1.1)",
+              },
+            }} 
           >
-            {isLoading ? "Sending..." : "Send"}
-          </Button>
```

Comment:
Change the icon here based on the `isLoading` state
Maybe use the [Sync Icon](https://mui.com/material-ui/material-icons/?query=sync&selected=Sync) to for when the `isLoading` state is true.

so something like this
```typescript
{isLoading ? <SyncIcon /> : <SendIcon />}
```